extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/BarrageUtil.java

```diff
@@ -657,10 +654,10 @@ private static ArrowType arrowTypeFor(Class<?> type) {
                     return Types.MinorType.LIST.getType();
                 }
                 if (type == LocalDate.class) {
-                    return LOCAL_DATE_TYPE;
+                    return Types.MinorType.DATEDAY.getType();
```

Comment:
Since we've switched formats, should we be be using `DATEMILLI`?
On the other hand, `DateMilliVector` seems to map to `LocalDateTime`, not `LocalDate`. Have we made a mistake?